bftengine/src/bftengine/NewViewMsg.hpp

```diff
@@ -36,6 +36,8 @@ namespace bftEngine
 
 			bool includesViewChangeFromReplica(ReplicaId replicaId, const Digest& viewChangeReplica) const;
 
+			const uint16_t elementsCount() const;
```

Comment:
Why do we need this?
It should always return 2f+2c+1

Reply:
But you still store it in NewViewMsgHeader . :-) and I just need access to this field.